### DISCUSSION TITLE
Confusing error message in SubsonicClient.php

### DISCUSSION THREAD

**Discussion by JOJ0** [ampache/ampache#3888] [Q&A] [answered]
What does this error message trying to tell me:

```
2024-03-19T10:56:15+00:00 [ampache] (log.lib) -> [Runtime Error] Trying to access array offset on value of type null in file /var/www/src/Module/Catalog/SubsonicClient.php(215)
```

**Comment by lachlan-00** [comment, marked as answer]
it means there wasn't a successful response form the server.

I'll put the response into the error debug so we can see what it does. probably time to build a docker image again after all the changes

![image](https://github.com/ampache/ampache/assets/1305249/5da3fb06-2bf8-4c20-b327-df7d07329088)
